java/rocksjni/portal.h

```diff
@@ -2498,6 +2528,24 @@ class JniUtil {
 
     return cvalue_len;
   }
+
+  template <class T>
```

Comment:
Documentation missing for this function.

Reply:
Also the naming seems odd as this has nothing to do with an Indirect ByteBuffer (instead it appears to work with a byte array)